Dockerfile

```diff
@@ -6,7 +6,20 @@ LABEL version="latest"
 ENV DEBIAN_FRONTEND noninteractive
 ENV APPDIR /app
 ENV LANG C.UTF-8
-ENV CONDA_VERSION 4.3.31
+
+# Versions, and expose labels for exernal usage
+ENV PYTHON_VERSION_27 2.7.13
+ENV PYTHON_VERSION_33 3.3.6
+ENV PYTHON_VERSION_34 3.4.7
+ENV PYTHON_VERSION_35 3.5.4
+ENV PYTHON_VERSION_36 3.6.2
+ENV CONDA_VERSION 4.4.10
+LABEL python.version_27 $PYTHON_VERSION_27
```

Comment:
From the official docs, it seems that we need the `=` sign here, similar to the LABEL from line 4, https://docs.docker.com/engine/reference/builder/#label

    LABEL python.version_27=$PYTHON_VERSION_27

Reply:
Ah good catch